docs/server-config.md

```diff
@@ -87,6 +87,8 @@ at [`logging.config` documentation page](https://docs.python.org/3.6/library/log
 - `TABPY_MAX_REQUEST_SIZE_MB` - maximal request size supported by TabPy server
   in Megabytes. All requests of exceeding size are rejected. Default value is
   100 Mb.
+- `TABPY_EVALUATE_ENABLE` - enable evaluate api. Default
```

Comment:
I might add a little to the desc here, ie 'enable evaluate api to execute ad-hoc Python scripts'